src/controllers/session/login/login.scss

```
@@ -24,3 +24,27 @@
         width: fit-content;
     }
 }
+
+.inputContainer {
+    position: relative;
+}
+
+.passwordToggle {
+    position: absolute;
+    width: 3em;
+    height: 3em;
+    top: 2.95em;
+    transform: translateY(-50%);
+    right: 0.1em;
+    background: none;
+    border: none;
+    cursor: pointer;
+    padding: 0.1em;
+    align-items: center;
+    justify-content: center;
+}
+
+.passwordToggle .material-icons {
+    font-size: 1.8em;
+    color: #666;
+}
```

`color` on the icon overrides the highlight when selected - move it to the button style:
```suggestion
    justify-content: center;
    color: #666;
}

.passwordToggle .material-icons {
    font-size: 1.8em;
}
```